src/neuroglancer/whats_new/whats_new.ts

```diff
@@ -24,15 +24,22 @@ const generateWhatsNew = (GHCommits: string[] = []) => {
 };
 
 export const findWhatsNew = async (viewer: Viewer) => {
-  let url =
-          `https://script.google.com/macros/s/AKfycbzVt6TLlJonmfU0EKTZVthi9pbM9dY1TYfTIH985tLUc8TZ5BNG/exec`,
-      WNCommits = JSON.parse(localStorage.getItem('WNCommits') || '[]'), headers = {
-        'Content-Type': 'text/plain;charset=utf-8',
-      },
-      body = JSON.stringify({
-        path: updateFName
-        // since: (WNCommits.length) ? WNCommits[0].commit.author.date : void(0)
-      });
+  const url =
+      `https://script.google.com/macros/s/AKfycbzVt6TLlJonmfU0EKTZVthi9pbM9dY1TYfTIH985tLUc8TZ5BNG/exec`;
+  const newUser = !localStorage.getItem('ng-newuser');
+  if (newUser) {
+    localStorage.setItem('ng-newuser', '1');
```

Comment:
We should be using cookies for these kinds of functions so power users and configure their setup and reset their browser if they switch off with a new user more easily.

Reply:
A better solution is to tie the flag to a User's account rather than the browser. That being said, apparently localStorage is more appropriate than cookies for purely client facing purposes. At least with Chrome Cookies and LocalStorage are considered the same in terms of how its managed

Reply:
The exact mechanism to use is up for debate, if we have server side uses for this flag we will probably put it in a cookie or tie it to user account